katharsis-core/src/main/java/io/katharsis/utils/ClassUtils.java

```diff
@@ -16,9 +16,6 @@
  */
 public class ClassUtils {
```

Comment:
There is a whole lost percent of coverage. That is somewhat troubling

On Fri, Sep 2, 2016, 8:11 AM SonarQube Reporter <notifications@github.com>
wrote:

> In katharsis-core/src/main/java/io/katharsis/utils/ClassUtils.java
> <https://github.com/katharsis-project/katharsis-framework/pull/104#discussion_r77362114>
> :
>
> > @@ -16,9 +16,6 @@
> >   */
> >  public class ClassUtils {
>
> [image: MAJOR]
> <https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png>
> Add a private constructor to hide the implicit public one. [image: rule]
> <https://sonarqube.com/coding_rules#rule_key=squid%3AS1118>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/katharsis-project/katharsis-framework/pull/104/files/4621aac399372494c3dcdb12e77d4947ccdd94d7#r77362114>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABaI0Dj3XhHOOat-2ssl7eEMUBnvMmIYks5qmDwogaJpZM4Jy4e0>
> .
>